frontend/index.html

```diff
@@ -5,6 +5,27 @@
 		<meta name="description" content="A marebling tool">
 		<link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
 		<link rel="stylesheet" href="stylesheet.css">
+		<script>
+			function switchStore() {
+				if(switchStore.input.checked) {
+					window.localStorage.setItem('store-active', '1');
+				} else {
+					if(window.confirm('When you the broswer storage all stored data will be cleard!\nthere is no way to undo this!')) {
```

Comment:
Maybe change to: When you disable browser storage all stored options will be cleared.\nThere is no way to undo this!

Reply:
But this is the reason why the user gets a dialog box to verify that he really wants to clear its data.
The ratio was that when you disable storage you don't want a clear setup and not you're old clutter :thinking: